src/scss/common/pages.scss

```
@@ -146,3 +146,8 @@ main {
     }
 }
 
+.context-picker-container {
+  @media (min-width: 992px) {
```

Ditto as to the above, I think something like:
```suggestion
  @include media-breakpoint-up(lg) {
```
is equivalent?